### PR TITLE
Update eslint and fix eslint violations

### PR DESCRIPTION
**Pull request by nabdelgadir**
### Description

Connect to https://github.com/strongloop-internal/scrum-apex/issues/425

### Checklist

- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

**Comment by nabdelgadir** [comment]
Oops duplicate of https://github.com/strongloop/loopback-connector-swagger/pull/58. Closing this.

**Comment by nabdelgadir** [comment]
CI: `No identified problem
No problems were identified. If you know why this problem occurred, please add a suitable Cause for it.`

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,3397 @@
+{
```
Comment (bajtos) [inline package-lock.json]: We are not using package-lock files in LB3 modules, please remove it.

File: package.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
     "swagger-parser": "^3.4.1"
   },
   "devDependencies": {
-    "eslint": "^3.14.1",
+    "eslint": "^4.18.2",
     "eslint-config-loopback": "^8.0.0",
```
Comment (bajtos) [inline package.json]: Can you also update the config to its latest version please?

Comment (bajtos) [reply package.json]:
```
    "eslint-config-loopback": "^13.1.0",
```

**Pull request merged**